Platform/OpenVariableRuntimeDxe/VariableDxe.c

```diff
@@ -648,10 +648,8 @@ VariableServiceInitialize (
   )
 {
   EFI_STATUS           Status;
-  UINTN                OffsetQVI;
-  UINTN                HeaderQVI;
-  EFI_EVENT            EndOfDxeEvent;
   EFI_EVENT            ReadyToBootEvent;
+  EFI_EVENT            EndOfDxeEvent;
```

Comment:
(These now match the original order, in the file from which this code was derived, again.)